app/build.gradle

```diff
@@ -5,15 +5,15 @@ apply plugin: 'kotlin-kapt'
 
 android {
 
-    compileSdkVersion 29
-    buildToolsVersion '29.0.2'
+    compileSdkVersion 31
```

Comment:
I can't build the project, Does it necessary to update kotlin & compileSdkVersion versions?